stl/inc/random

```diff
@@ -1863,33 +1865,34 @@ public:
     struct param_type { // parameter package
         using distribution_type = uniform_int;
 
-        param_type() {
+        param_type() noexcept /* strengthened */ {

```

Comment:
For `uniform_int::param_type` and `uniform_real::param_type`, their constructors shouldn't be marked as `strengthened`, as these are non-Standard TR1 types and the constructors aren't inherited by the Standard machinery `uniform_int_distribution::param_type` and `uniform_real_distribution::param_type`.